addons/mail/static/src/core/common/chat_hub_service.js

```diff
@@ -0,0 +1,10 @@
+import { ChatHub } from "@mail/core/common/chat_hub";
+import { registry } from "@web/core/registry";
+
+export const chatHubService = {
+    dependencies: ["mail.store"],
```

Comment:
To be fully correct, it depends on all these 3:
```js
this.store = useService("mail.store");
        this.ui = useService("ui");
this.busMonitoring = useService("bus.monitoring_service");
```
It's best to list them all, even if they others are already ready.

Reply:
Others are supported in the `mail.store` dependencies. `ui` is directly in its dependencies and `bus.monitoring_service` is deployed by `bus_service`.